src/internal_events/common.rs

```diff
@@ -150,14 +150,13 @@ impl<'a, const INTENTIONAL: bool> InternalEvent for ComponentEventsDropped<'a, I
 
 #[derive(Debug)]
 pub struct SinkRequestBuildError<E> {
```

Comment:
:thought_balloon: Wondering if it would be helpful to have a comment indicating that why we are not specifying a name, so our future selves will not have to rely on long term memory :)

Reply:
Sure, can do.

Reply:
Resolved in https://github.com/vectordotdev/vector/pull/14510/commits/a4600105f22e9b7b13ff262740a5067ae490cf14